rust/tw_keypair/src/ecdsa/secp256k1/private.rs

```diff
@@ -81,6 +81,7 @@ impl<'a> TryFrom<&'a str> for PrivateKey {
 
 impl ToBytesZeroizing for PrivateKey {
     fn to_zeroizing_vec(&self) -> Zeroizing<Vec<u8>> {
+        // TODO: Why do this twice like this?
         let secret = Zeroizing::new(self.secret.to_bytes());
         Zeroizing::new(secret.as_slice().to_vec())
```

Comment:
That's because `k256::ecdsa::SigningKey` doesn't provide a method to return `Vec<u8>` directly.
Instead, it returns `FieldBytes<Secp256k1>` that is not zeroized on drop. In terms of security, we should zeroize any memory regions containing secret keys even if they are allocated on stack